Mars-QA/StepsDefinition/StepDefinitions.cs

```diff
@@ -0,0 +1,209 @@
+using Aspose.Cells.Drawing;
+using Mars_QA.Pages;
+using Mars_QA.Utilis;
+using Microsoft.VisualStudio.TestPlatform.CommunicationUtilities;
+using NUnit.Framework;
+using OpenQA.Selenium.Chrome;
+using System;
+using System.Collections.Generic;
+using TechTalk.SpecFlow;
+
+namespace Mars_QA.StepsDefinition
+{
+    [Binding]
+    public class StepDefinitions : CommonDriver
+    {
+        Login loginPageObj;
+        Profile profileObj;
+        Edit editObj;
+        Delete deleteObj;
+
+        public string ActualSkillMessage { get; private set; }
+
+        public StepDefinitions()
+        {
+            loginPageObj = new Login();
+            profileObj = new Profile();
+            editObj = new Edit();
+            deleteObj = new Delete();
+        }
+
+        [Given(@"I Logged into a portal")]
+        public void GivenILoggedIntoAPortal()
+        {
+            // Open Chrome Browser
+            Initialize();
```

Comment:
so Initialize should not be called like this you should make use of hooks

Reply:
https://docs.specflow.org/projects/specflow/en/latest/Bindings/Hooks.html

Reply:
ok I will make changes .